arrows/core/transform.cxx

```diff
@@ -193,7 +194,9 @@ necker_reverse(vital::camera_map_sptr& cameras,
   vital::rotation_d Rz180(vital::vector_4d(0.0, 0.0, 1.0, 0.0));
   for(cam_map_t::value_type& p : cams)
   {
-    vital::simple_camera* flipped = new vital::simple_camera(*p.second);
```

Comment:
this code was like from before I knew about `make_shared`, so it could be reworked to avoid the raw pointer and `new` call.

Reply:
Since the flipped camera is written back to the camera map is there any reason for a copy to be made or can we just grab a pointer and change the camera in place.

Reply:
I think it's safer to make a copy.  The way this function is often used is to keep a pointer to the original camera map and then pass a pointer to that camera map into this function, which returns a new reversed camera map its place.  We then often use both the original cameras and the reverse cameras to compare and see which is better.  If we change the cameras in place it will corrupt our copy of the original cameras.   Of course we could work around this by requiring the user to explicitly clone the camera map before passing to this function, but I think it's easier to just do the copy in this function until we have a reason to do otherwise.